oslabs-beta/Svelvet
# Mayson/canvaskeyboard func

Description
This pull request provides minor changes to the canvas' camera movement, and adds functionality that allows movement between nodes through just the use of the keyboard along with increased interactivity.
We can now move in between nodes by using option + tab.

Trello Task
Selecting nodes on canvas through keyboard

Testing Instructions
Run npm run dev to start up the dev server. 
Click on the canvas and use the arrow keys to move the camera. The camera now no longer moves in an inverted fashion.
On the keyboard, press option + tab to cycle through the nodes. (Just pressing tab allowed focus on the nodes but it did not allow movement. Using option + tab allows for movement.)

Checklist
N/A

Additional Notes, Images, etc.

>

>

Co-authored-by: Isaac Bocage <[ibocage1@gmail.com](mailto:ibocage1@gmail.com)>

Co-authored-by: Julian Bos <[julianb1212@gmail.com](mailto:julianb1212@gmail.com)>

Co-authored-by: Solomon Moon <[chochobo91@gmail.com](mailto:chochobo91@gmail.com)>
